### PR TITLE
Update + sync getting started scripts for SC

### PR DESCRIPTION
**Pull request by fhildeb**
Updates and simplifies Hardhat deployment scripts based on playground changes:

Tested snippets from:
- [x] https://github.com/lukso-network/lukso-playground/pull/36

### CODE REVIEW

File: docs/learn/smart-contract-developers/_category_.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-label: '🧙‍♂️ Working with Smart Contracts'
+label: '🧙‍♂️ Working with Contracts'
```
Comment (Hugoo) [inline docs/learn/smart-contract-developers/_category_.yml:1]: Why?

I think it is god to keep "smart contracts" (SEO, etc.)

**Review by Hugoo — COMMENTED** [review]
not sure about duplicated content, maybe hving it at only place is good?

Just sharing thoughts here

File: docs/learn/smart-contract-developers/_category_.yml (hunk 1 above)
Comment (fhildeb) [reply docs/learn/smart-contract-developers/_category_.yml:1]: It's the only chapter that does not fit within one line 😅 Maybe do `Build Smart Contracts` or `Create Smart Contracts`? WDYT?

img1

**Comment by fhildeb** [comment]
I get your point regarding the duplicated content from the LSP7 token creation guide. From my side, it's useful to grasp the flow for deploying and using LSPs within Hardhat. 

When you go from `Getting started` -> `Create LSP7` (or any other LSP) you immediately get the hang on what you are doing and why it needs to be in this order.

**Review by Hugoo — APPROVED** [review]

**Pull request merged**